zcash_primitives/src/transaction/components/tze.rs

```diff
@@ -2,15 +2,14 @@
 #![cfg(feature = "zfuture")]
 
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
-
```

Comment:
Slight misunderstanding of my previous comment: by our existing convention, `use byteorder` would be grouped with `use zcash_encoding`.